streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/desc/ListStateDescriptor.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ray.streaming.state.keystate.desc;
+
+
+import static org.ray.streaming.state.config.ConfigKey.DELIMITER;
+import org.ray.streaming.state.keystate.state.ListState;
+
+/**
+ * ListStateDescriptor.
+ */
+public class ListStateDescriptor<T> extends AbstractStateDescriptor<ListState<T>, T> {
+
+  private final boolean isOperatorList;
+  private int index;
+  private int number;
```

Comment:
What does `number` mean?

Reply:
It's used in the operator state meaning the partitionNum. I'll try to make it more straightforward

Reply:
Thanks for your update!